data/docker-compose-data_management.yml

```diff
@@ -20,6 +20,22 @@ services:
         - API_POSTGRES_USER=marxan-api
         - API_POSTGRES_PASSWORD=marxan-api
         - API_POSTGRES_DB=marxan-api
+
+  marxan-restore-data:
+    build:
+        context: ./data_restore
+        dockerfile: Dockerfile
+    user: "$UID:$GID"
+    command: make
+    environment:
+        - POSTGRES_API_SERVICE_PORT=3432
+        - POSTGRES_GEO_SERVICE_PORT=3433
+        - GEO_POSTGRES_USER=marxan-geo-api
+        - GEO_POSTGRES_PASSWORD=marxan-geo-api
+        - GEO_POSTGRES_DB=marxan-geo-api
+        - API_POSTGRES_USER=marxan-api
+        - API_POSTGRES_PASSWORD=marxan-api
+        - API_POSTGRES_DB=marxan-api
 networks:
     default:
       external:
```

Comment:
we can also remove this later on (unless we are using this specifically for data containers) - I have removed the explicit network config from the other compose files